src/fetchers/stats-fetcher.js

```diff
@@ -123,7 +126,10 @@ async function fetchStats(
     rank: { level: "C", score: 0 },
   };
 
-  let res = await retryer(fetcher, { login: username });
+  let res = await retryer(fetcher, { 
+    login: username, 
+    starttime: year ? `${year}-01-01T00:00:00Z` : undefined
+  });
 
   if (res.data.errors) {
     logger.error(res.data.errors);
```

Comment:
Could you also please add documentation to [the README](https://github.com/anuraghazra/github-readme-stats/blob/master/readme.md) and add some tests?